client/src/pages/checkout/Checkoutpage.tsx/account-form.module.css

```diff
@@ -0,0 +1,13 @@
+.account-form {
+    display: flex;
+    flex-direction: column;
+}
+
+.input {
+    margin: .4rem;
+    color: grey;
+}
+
+/* .input:invalid {
+    color: red;
+} */
```

Comment:
commented out section - is it needed?

Reply:
Added it back in. When a user adds an email that's not valid, the text will be red.